Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,46 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+```
+a)  a = 0
+    while (a < n * n * n):==> O(n)
+      a = a + n * n ========> O(1)
+```
+
+`This is O(n) because every time n is incresed the run time grows.`
+
+```
+
+b)  sum = 0
+    for i in range(n): =======================> O(n)
+      i += 1 =================================> O(1)
+      for j in range(i + 1, n): ==============> O(n)
+        j += 1 ===============================> O(1)
+        for k in range(j + 1, n): ============> O(n)
+          k += 1 =============================> O(1)
+          for l in range(k + 1, 10 + k): =====> O(n)
+            l += 1 ===========================> O(1)
+            sum += 1 =========================> O(1)
+```
+

```

Comment:
heheheheh, awesome job here. almost like you got some pro tutoring at the beginning of the week or something!